shared/fs/common/path-item-action/layout.js

```diff
@@ -77,8 +102,8 @@ const getRawLayout = (path: Types.Path, pathItem: Types.PathItem, me: string): L
         showInSystemFileManager: !isMobile,
         // share menu items
         // eslint-disable-next-line sort-keys
-        sendAttachmentToChat: flags.sendAttachmentToChat && isMobile && pathItem.type === 'file', // desktop uses separate button
-        sendLinkToChat: isMobile && Constants.canSendLinkToChat(parsedPath), // desktop uses separate button
+        sendAttachmentToChat: flags.sendAttachmentToChat && pathItem.type === 'file', // desktop uses separate button
```

Comment:
Is removing `isMobile` accidental, or is the comment now a lie?

Reply:
Comment is now a lie and I got rid of them in https://github.com/keybase/client/pull/17164